sigcomm-experiment/env-setup/200-k8s_install.sh

```diff
@@ -71,6 +100,10 @@ function deploy_k8s_master {
 #	kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml
  	kubectl apply -f https://gist.githubusercontent.com/ShixiongQi/f56db40853965090dd2d6cf723ebd8b3/raw/e45eab1722d37255382d21f57ce48ecbd9fe3d3e/y-calico-tigera-operator.yaml
  	kubectl apply -f https://gist.githubusercontent.com/ShixiongQi/f56db40853965090dd2d6cf723ebd8b3/raw/e45eab1722d37255382d21f57ce48ecbd9fe3d3e/y-calico-custom-resources.yaml
```

Comment:
Hi @glitchkyle The yaml file that installs calico CNI can be removed. This seems to cause conflicts with the weave CNI and crashes the control plane components on the master node (Please see the screenshot).
<img width="2518" alt="Screenshot 2024-09-15 at 10 40 03 AM" src="https://github.com/user-attachments/assets/462243c1-44c4-4627-bd2c-b38c4ed47efb">